### PR TITLE
Update readme to include information about new repo structure

### PR DESCRIPTION
**Pull request by rocodes**
## Status

Ready for review

## Description of Changes

Update README "What's in this repo" section

Changes proposed in this pull request:

readme-only 

## Testing

- [ ] Visual/content review plz
- [ ] lint is happy 

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing pilot instances
2. New installs

### If you have made changes to the provisioning logic

- [ ] All tests (`make test`) pass in `dom0`

### If you have added or removed files

- [ ] I have updated `MANIFEST.in` and `rpm-build/SPECS/securedrop-workstation-dom0-config.spec`

### If documentation is required

- [ ] I have opened a PR in the [docs repo](https://github.com/freedomofpress/securedrop-workstation-docs) for these changes, or will do so later
- [ ] I would appreciate help with the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,16 +73,34 @@ See below for a closer examination of this process, and see `docs/images` for sc
...
+- [SecureDrop Export](https://github.com/freedomofpress/securedrop-client/tree/main/export#readme) is installed in `sd-devices` and is used to manage printing and exporting files.
+- There are other components as well: the *SecureDrop Client* opens all submissions in the `sd-viewer` disposable VM; and a logging VM, `sd-log`, is provisioned to capture logs locally from various parts of the system; and a [Whonix](https://www.whonix.org/wiki/Homepage) VM, `sd-whonix`, is provisioned with instance-specific information required to access the authenticated onion service used by journalists.
+
+### Additional Notes:
 - `files/config.json.example` is an example config file for the provisioning process. Before use, you should copy it to `config.json` (in the repository's root directory), and adjust to reflect your environment.
 - `sd-journalist.sec.example` is an example GPG private key for use in decrypting submissions. It must match the public key set on a SecureDrop server used for testing. Before use, you should copy it to `sd-journalist.sec`, or store the submission key used with your SecureDrop server as `sd-journalist.sec`.
```
Comment (zenmonkeykstop) [inline README.md:103]: I'd be stronger here and say something like "it is the public key component of the GPG keypair used in testing and development of SecureDrop - this keypair must not be used in a production environment." Maybe less grumpily.

File: README.md [hunk 2]
```diff
@@ -73,16 +73,34 @@ See below for a closer examination of this process, and see `docs/images` for sc
...
+in individual VMs once they have been provisioned.
 - The [SecureDrop Client](https://github.com/freedomofpress/securedrop-client) is installed in `sd-app` and will be used to access the SecureDrop server *Journalist Interface* via the SecureDrop proxy.
 - The [SecureDrop Proxy](https://github.com/freedomofpress/securedrop-client/tree/main/proxy#readme) is installed in `sd-proxy` to communicate to the SecureDrop server *Journalist Interface* via `sd-whonix`.
-- Within `sd-app`, the *SecureDrop Client* will open all submissions in the `sd-viewer` disposable VM.
+- [SecureDrop Export](https://github.com/freedomofpress/securedrop-client/tree/main/export#readme) is installed in `sd-devices` and is used to manage printing and exporting files.
+- There are other components as well: the *SecureDrop Client* opens all submissions in the `sd-viewer` disposable VM; and a logging VM, `sd-log`, is provisioned to capture logs locally from various parts of the system; and a [Whonix](https://www.whonix.org/wiki/Homepage) VM, `sd-whonix`, is provisioned with instance-specific information required to access the authenticated onion service used by journalists.
```
Comment (zenmonkeykstop) [inline README.md:99]: You could just enumerate these components separately in the same way as the previous two, they have their own (meta)packages etc even if they're not as complex.

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]
Some suggestions for edits but overall great, thanks for grabbing this!

**Comment by rocodes** [comment]
@zenmonkeykstop addressed feedback in https://github.com/freedomofpress/securedrop-workstation/pull/1072/commits/f0bea6e848d020e6eaafdf56fa9746904c223183

**Comment by rocodes** [comment]
CI is unrelated

**Review by zenmonkeykstop — APPROVED** [review]
LGTM

**Pull request merged**